pkg/systemd/overview-cards/systemInformationCard.jsx

```diff
@@ -113,28 +113,28 @@ export class SystemInfomationCard extends React.Component {
                 <CardTitle>{_("System information")}</CardTitle>
                 <CardBody>
                     <table className="pf-v5-c-table pf-m-grid-md pf-m-compact">
-                        <tbody>
-                            {this.state.hardwareText && <tr>
-                                <th scope="row">{_("Model")}</th>
-                                <td>
+                        <tbody className="pf-v5-c-table__tbody">
+                            {this.state.hardwareText && <tr className="pf-v5-c-table__tr">
+                                <th className="pf-v5-c-table__th" scope="row">{_("Model")}</th>
+                                <td className="pf-v5-c-table__td">
                                     <div id="system_information_hardware_text">{this.state.hardwareText}</div>
                                 </td>
                             </tr>}
-                            {this.state.assetTagText && <tr>
-                                <th scope="row">{_("Asset tag")}</th>
-                                <td>
+                            {this.state.assetTagText && <tr className="pf-v5-c-table__tr">
+                                <th className="pf-v5-c-table__th" scope="row">{_("Asset tag")}</th>
+                                <td className="pf-v5-c-table__td">
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19111-20230720-095528-59ce1183-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)